packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -196,7 +203,7 @@ contract L2ToL2CrossDomainMessenger is ISemver, TransientReentrancyAware {
         _storeMessageMetadata(source, sender);
 
         bool success;
-        (success, returnData_) = target.call{value: msg.value}(message);
+        (success, returnData_) = target.call{ value: msg.value }(message);
```

Comment:
my only question is, are we losing anything from removing `SafeCall`? why were we using it in the first place?

Reply:
The reason behind using `SafeCall` was not copying to memory the return data, to avoid a return data bomb attack vector where the taget contract returns a huge amount of data that consumes the tx's gas and hence reverts.

Has this discussion already taken place?